Services/CO.CDP.OrganisationInformation/Qualification.cs

```diff
@@ -0,0 +1,10 @@
+namespace CO.CDP.OrganisationInformation;
+public class OrganisationQualification
+{
+    public int Id { get; set; }
+    public string? AwardingByPersonOrBodyName { get; set; }
+    public DateTimeOffset DateAwarded { get; set; }
+    public string? Name { get; set; }
+    public DateTimeOffset CreatedOn { get; set; }
+    public DateTimeOffset UpdatedOn { get; set; }
+}
```

Comment:
It doesn't look like this class needs to be in the shared CO.CDP.OrganisationInformation package. At least, let's not start with putting it here in the first place.